tests/system/Makefile

```diff
@@ -28,7 +28,7 @@ test:
       --stdout all \
       --stdout-inline \
       --ssl-no-verify \
-      --timeout 360000 \
+      --timeout 1800 \
```

Comment:
I don't understand why did we made the timeout smaller, what is this fixing? Thanks

Reply:
yes it is fixing something... details in the jira... but the outer timeout using the unix `timeout` was 2 hrs... first the cluster expires in 2 hrs... if we wait that long we can't get a diagnostic bundle.  second that is all or nothing.   This change ensures that each individual test does not go longer than 30 mins.   If there is 1 long running test that is causing these issues (which is what is suspected)... then that test will not run for 1-2 hrs... it will be killed in 30 mins and we move on.   
We are also changing the timeout because whoever originally assigned the time likely didn't understand that this isn't in milliseconds... it is seconds.    Do we really want a timeout that is 100 hours?